lib/HLSL/DxilValidation.cpp

```diff
@@ -2199,8 +2203,11 @@ static void ValidateResourceDxilOp(CallInst *CI, DXIL::OpCode opcode,
     default:
       ValCtx.EmitInstrError(CI,
                             ValidationRule::InstrResourceKindForTextureLoad);
-      break;
+      return;
```

Comment:
As above, return prevents attempts to validate invalid resources as an error is already produced